js/utils/constants.js

```diff
@@ -90,6 +90,7 @@
         COMPACT_SELECT_ASSOCIATION_LINK: 'compact/select/association/link',
         COMPACT_SELECT_ASSOCIATION_UNLINK: 'compact/select/association/unlink',
         COMPACT_SELECT_FOREIGN_KEY: 'compact/select/foreign_key',
+        COMPACT_SELECT_BULK_FOREIGN_KEY: 'compact/select/foreign_key/bulk',
```

Comment:
I figured having `bulk` at the end would be consistent with our other `contexts` and allows for proper inheritance of contexts. 

To reiterate from the other comment, using `multiselect` would be inconsistent with the other contexts we have with `select` in them. This also breaks inheritance. That's also why `bulk` (or `multi`) should be at the end of the string.